src/data/notification/__test__/__snapshots__/notification.test.js.snap

```diff
@@ -1,27 +1,3 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
```

Comment:
@vinay3206 why is the snapshot empty?

Reply:
@maartenhemmes snapshot got updated because response is empty.

Actually, u can not always expect the same notifications in the response so snapshot needs to be kept updated